src/codecoverage/backend/codecoverage_backend/gcp.py

```diff
@@ -241,18 +241,18 @@ def find_closest_report(self, repository, changeset):
             self.ingest_pushes(repository, min_push_id=push_id-1, nb_pages=1)
 
         # Load report from that push
-        return self.find_report(repository, min_push_id=push_id)
+        return self.find_report(repository, max_push_id=push_id, min_push_id='+inf', reverse=False)
```

Comment:
It's a bit confusing that max_push_id < min_push_id

Reply:
Reverse order by default to get the history & latest easily